src/GUI/Shared/NavMenu.razor.css

```diff
@@ -1,62 +1,24 @@
-.navbar-toggler {
-    background-color: rgba(255, 255, 255, 0.1);
-}
+.nav-menu {
+    width: 21%;
+    height: 850px;
```

Comment:
maybe it will be better if you use relative values for height like "height: 100vh;" and absolute values for width, because when the screen will be smaller the layout will break

Reply:
Good solution for height. But for width better use percent